ui/components/component-library/component-library-components.scss

```diff
@@ -4,6 +4,12 @@
 * This will help improve specificity and reduce the chance of
 * unintended overrides.
 **/
+
+// Temp CSS Variables
+:root {
+  --opacity-disabled: 0.5;
+}
+
```

Comment:
Ooh, interesting. Should we set aside some time to compare Sass vars vs CSS vars? Another option is to create a stylesheet for opacity similar to z-index.css.

Reply:
That makes sense. I'll create a ticket to add opacity tokens